### PR TITLE
First support for Minerva LX

### PR DESCRIPTION
**Pull request by archetype-made**
## Description
Adds support for Minerva LX by Archetype.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/archetype/minervalx/config.h [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+#pragma once
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+
+#define WS2812_PIO_USE_PIO1
+
+#define RGBLIGHT_DEFAULT_MODE RGBLIGHT_MODE_RAINBOW_SWIRL
+
```
Comment (waffle87) [inline keyboards/archetype/minervalx/config.h:24]:
```suggestion
```

File: keyboards/archetype/minervalx/info.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "backlight": true
```
Comment (waffle87) [inline keyboards/archetype/minervalx/info.json:14]:
```suggestion
        "bootmagic": true,
        "extrakey": true,
        "mousekey": true,
        "backlight": true
```

File: keyboards/archetype/minervalx/readme.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Minerva LX
+
+![Minerva LX](https://archetypemade.com/minerva-is-here/)
```
Comment (waffle87) [inline keyboards/archetype/minervalx/readme.md:3]: This isn't an image link...

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/archetype/minervalx/config.h (hunk 1 above)
Comment (archetype-made) [reply keyboards/archetype/minervalx/config.h:24]: Aren't these lines needed for the WS2812 Driver?

Comment (waffle87) [reply keyboards/archetype/minervalx/config.h:24]: Are you using the WS2812 driver?

File: keyboards/archetype/minervalx/readme.md (hunk 3 above)
Comment (archetype-made) [reply keyboards/archetype/minervalx/readme.md]: I added the image link in the commit b470462

File: keyboards/archetype/minervalx/config.h (hunk 1 above)
Comment (archetype-made) [reply keyboards/archetype/minervalx/config.h:24]: You're right! No need for the driver as we're only using Through-Hole UV LEDs.

**Review by waffle87 — APPROVED** [review]

File: keyboards/archetype/minervalx/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    
+    [0] = LAYOUT(
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_RSFT,
+        KC_LCTL, KC_LGUI, KC_LALT,          KC_SPC,           KC_SPC,           KC_SPC,           KC_RALT, KC_RGUI, MO(1),   KC_RCTL
+    )
```
Comment (zvecr) [inline keyboards/archetype/minervalx/keymaps/default/keymap.c:27]: `MO(1)` here would go out of bounds
```suggestion
    ),
    [1] = LAYOUT(
        QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______,          _______,           _______,         _______,          _______, _______, _______, _______
    )
```

File: keyboards/archetype/minervalx/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    
+    [0] = LAYOUT(
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_RSFT,
+        KC_LCTL, KC_LGUI, KC_LALT,          KC_SPC,           KC_SPC,           KC_SPC,           KC_RALT, KC_RGUI, MO(1),   KC_RCTL
+    )
```
Comment (zvecr) [inline keyboards/archetype/minervalx/keymaps/via/keymap.c:27]: While strictly not required due to dynamic keymap being enabled, might as well align the keymaps
```suggestion
    ),
    [1] = LAYOUT(
        QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______,          _______,           _______,         _______,          _______, _______, _______, _______
    )
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/archetype/minervalx/info.json
Comment (fauxpark) [inline keyboards/archetype/minervalx/info.json:1]: Please run `qmk format-json -i` on this file.

File: keyboards/archetype/minervalx/readme.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+* Hardware Supported: RP2040-based proprietary PCB.
+* Hardware Availability: https://archetypemade.com/minerva-is-here/
+
+Make example for this keyboard (after setting up your build environment):
+
+	make archetype/minervalx:default
+
+Flashing example for this keyboard:
+
+	make archetype/minervalx:default:flash
```
Comment (fauxpark) [inline keyboards/archetype/minervalx/readme.md:17]:
```suggestion
    make archetype/minervalx:default

Flashing example for this keyboard:

    make archetype/minervalx:default:flash
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/archetype/minervalx/info.json
Comment (Gondolindrim) [reply keyboards/archetype/minervalx/info.json:1]: Done in [1e26dd1](https://github.com/qmk/qmk_firmware/pull/23291/commits/1e26dd1a4d358b2c89f4cb6e5ec0a4cfce36527a)

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**